pygenie/jobs/core.py

```diff
@@ -203,6 +204,15 @@ def _add_dependency(self, dep):
         if dep not in self._dependencies:
             self._dependencies.append(dep)
 
+    def _add_config(self, config):
```

Comment:
looks like... python voodo... someone better at python like @charsmith or @irontable would be better to answer. I just copied what was done for job dependencies.